test/Altinn.AccessManagement.Tests/Altinn.AccessManagement.Tests.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="JWTCookieAuthentication" Version="3.0.0" />
+    <PackageReference Include="JWTCookieAuthentication" Version="3.0.1" />
```

Comment:
Test should get all it's required (non test specific) dependencies through the accessmanagement project reference